src/components/folder/Modal.js

```diff
@@ -0,0 +1,156 @@
+import '../../style/modal.css';
+import modalClose from '../../images/modalClose.svg';
+import modalFacebook from '../../images/modalFacebook.svg';
+import modalLink from '../../images/modallink.svg';
+import modalKakao from '../../images/modalKakao.svg';
+import check from '../../images/check.svg';
+import { useState } from 'react';
+
+function Modal({ values, onClose, folderName, folders, userId }) {
+  const [selectedFolderId, setSelectedFolderId] = useState(null);
+  const closeModal = () => onClose(false);
+  const { url } = values;
+
+  const handleFolderButtonClick = (folderId) => {
```

Comment:
```suggestion
  const changeSelectedFolderId = (folderId) => {
```

어떤 이벤트에 동작하는지를 함수에 명시하기보다 어떤 행동을 하는지 명시하는게 어떨까요?
함수를 호출하는 문에서 `handleFolderButtonClick`이라고 되어 있으면 어떤 이벤트에 동작하는지는 알지만 어떤 행동을 하는지는 예측할 수 없게 됩니다.

반면 `changeSelectedFolderId`라고 명명하게 되면 로직을 모르더라도 최소한 `selectedFolderId`를 바꾸는 함수라고 예측할 수 있게되고 어떤 이벤트에 동작하는지는 바인딩된 이벤트를 확인하면 되기 때문에 어떤 이벤트에 어떤 동작을 하는지 예측하기 쉬워지는 장점이 있습니다!